### DISCUSSION TITLE
[Bug]: Can't generate PDF on Mac

### DISCUSSION THREAD

**Discussion by sam32019** [spatie/laravel-pdf#163] [Q&A]
### What happened?

You are unable to generate a PDF on Mac, due to the spatie/browsershot dependency requiring puppeteer through NPM.
This, however, is listed nowhere in the docs.

### How to reproduce the bug

- Have MacOS
- Run the code

### Package Version

1.5.1

### PHP Version

8.3.7

### Laravel Version

11.0

**Comment by talelmishali** [comment]
It’s most probably the underlying browsershot package. Make sure the node path is set correctly. Look at the docs on how to set the binaries path:  https://spatie.be/docs/browsershot/v4/requirements

Try to share some code/error so troubleshooting would be somewhat less of a guess of what might be the problem.

**Comment by FoxRocks76** [comment]
I'm running a Macbook Pro M1 and I've able to get it running this morning with just the instructions in the docs. Laravel 11, PHP 8.2, NPM 9.6.5.
